### PR TITLE
Update QA section README.md

### PR DESCRIPTION
**Pull request by lindalumitchell**
Added: Vision, channels, list of subsections.

Actual links to subsections will be added when those pages have been created / populated.

### CODE REVIEW

File: operations/research-and-development/quality/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+Some QA people are members of the core QA team, and others are part of feature or platform teams. All are part of the QA Guild. 
+
+If you'd like to learn more, you are welcome to join the Mattermost community and engage with QA and others in channels such as these:
+- [QA: Quality Assurance](https://community.mattermost.com/core/channels/quality-assurance) for general information and questions
+- [QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings) and the associated Board for info about our weekly Guild meetings
+- [QA: UI Test Automation](https://community.mattermost.com/core/channels/ui-test-automation) for info and discussions about test automation
+- [QA: Contributors](https://community.mattermost.com/core/channels/qa-contributors) for info and discussions about contributing to QA efforts at Mattermost
+
+## In this Section
+### [e2e Test Writing](https://developers.mattermost.com/contribute/webapp/end-to-end-tests)
```
Comment (saturninoabril) [inline operations/research-and-development/quality/README.md]:
```suggestion
### [E2E Test Writing](https://developers.mattermost.com/contribute/webapp/end-to-end-tests)
```

**Review by saturninoabril — DISMISSED** [review]
Thanks Linda! All good except for one request.

File: operations/research-and-development/quality/README.md (hunk 1 above)
Comment (lindalumitchell) [reply operations/research-and-development/quality/README.md]: Thanks! Made the change.

**Review by saturninoabril — APPROVED** [review]
Thanks Linda!

**Review by lindy65 — APPROVED** [review]
Thanks @lindalumitchell - looks good to me 👍

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**